packages/clay-css/src/scss/cadmin/variables/_globals.scss

```diff
@@ -329,8 +329,9 @@ $cadmin-component-transition: color 0.15s ease-in-out,
 
 $cadmin-component-focus-box-shadow: #{0 0 0 2px $cadmin-white,
 	0 0 0 4px $cadmin-primary-l0} !default;
-$cadmin-component-focus-inset-box-shadow: #{inset 0 0 0 2px $cadmin-primary-l0,
-	inset 0 0 0 4px $cadmin-white} !default;
+$cadmin-component-focus-inset-box-shadow: #{inset 0 0 0 0.125rem
```

Comment:
We shouldn't change to `rem` value here. How it works is if you change the `font-size` on the `html` element, the value will scale based on it. In cadmin, we are trying to make it hard to overwrite admin styles via site styles. This is why we use `px` values.